HW04.Rmd

```diff
@@ -1,126 +1,306 @@
+
+#print out using kable()
+kable(female_avg_age, format = "markdown",
+      col.names = c("Sex", "Average age of Deaths"),
+      caption = "Average age of all female gun related deaths from 2012-2014")
+```
+
+
+## How many white males with at least a high school education were killed by guns in 2012?
+
+```{r education}
+#create new data frame
+white_m_hs_ed <- gun_deaths %>%
+    filter(sex == "M", 
+           race == "White", 
+           education %in% c('HS/GED', 'Some college', 'BA+'),
+           year ==2012) %>%
+    count()
+
+#print out using kable()
+kable(white_m_hs_ed, format = "markdown",
+      col.names = "Number of Deaths",
+      caption = "Total number of white male gun related deaths with at least a high school education in 2012 ")
+```
+
+
+## Which season of the year has the most gun deaths?
+    ANSWER: From the table printed the season with the highest number of gun deaths is Summer.
+
+Assume that:
+
+* Winter = January-March
+* Spring = April-June
+* Summer = July-September
+* Fall = October-December
+
+```{r season}
+#create new data frame, 
+    #fct_collapse: Collapse factor levels into manually defined groups
+gun_deaths_season <- gun_deaths %>%
+    mutate(season = fct_collapse(.f = month,
+                                 Winter = c("Jan", "Feb", "Mar"),
+                                 Spring = c("Apr", "May", "Jun"),
+                                 Summer = c("Jul", "Aug", "Sep"),
+                                 Autumn = c("Oct", "Nov", "Dec"))) %>%
+    count(season) %>%  # count number of deaths per season
+    arrange(desc(n))
+
+#print out using kable()
+kable(gun_deaths_season, format = "markdown",
+      col.names = c("Season", "Number of Deaths"),
+      caption = "Total number of gun related deaths per season")
+```
+
+
+# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
+
+### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
+
+```{r race}
+#filter data
+filter_race <- gun_deaths %>%
+    filter(race %in% c("White", "Black", "Hispanic"),
+           intent %in% c("Suicide", "Homicide"))
+
+#create plot
+ggplot(filter_race, aes(race, fill = intent)) +
+    geom_bar(position = "dodge") +
+    scale_y_continuous(breaks = seq(0, 60000, by = 10000)) +
+    
+#add labels
+    labs(title = "Number of Gun Deaths by Race and Intent from 2012-2014",
+         x     = "Race",
+         y     = "Number of Deaths") 
+```
+
+(hint maybe looking at percentages would be good)
+```{r race_2}
+#find percentage
+new_filter_race<- filter_race %>%
+    select(race, intent) %>%
+    group_by(race, intent) %>%
+    count(race) %>%
+    group_by(race) %>%
+    mutate(percent = 100*n / sum(n)) #new varaible to show the percentage of deaths within a race
+
+kable(new_filter_race, format = "markdown",
```

Comment:
I noticed that you used kable to print out the table for almost every graph and I thought that was splendid of you to practise doing that! I only used kable to print out the very first table for this assignment, I wasn't sure if I was supposed to do it for every question. Now that I look back, it would have still been a good idea to do it for every question as it helps to visualize things.